stubs/pywin32/win32/win32wnet.pyi

```diff
@@ -29,7 +29,8 @@ def WNetGetLastError() -> tuple[Incomplete, Incomplete, Incomplete]: ...
 def WNetGetResourceParent(NetResource: _win32typing.PyNETRESOURCE, /) -> _win32typing.PyNETRESOURCE: ...
 def WNetGetConnection(connection: str | None = ..., /) -> str: ...
 
-NCB = _win32typing.PyNCB
-NCBType = _win32typing.PyNCB
 NETRESOURCE = _win32typing.PyNETRESOURCE
+NCB = _win32typing.PyNCB
+# old "deprecated" names, before types could create instances.
```

Comment:
What's weird is that it seems to be the other way around everywhere else. Where `ClassType` is the reference type, and `Class` is a method that creates an instance of `PyClass`.
The exception to the rule I guess. 🤷